.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+    "recommendations": [
+        "twxs.cmake",
+        "bradymholt.pgformatter"
```

Comment:
Let's not commit commit custom editor configs in here. We will all have custom configs and we can each maintain those.
for pgformatter, we should have a `make` target also.

Reply:
> for pgformatter, we should have a make target also.

I added a make target! It's called `format-sql`. Should I mention it in future docs?

The flow I plan to follow is that CI/CD will check if relevant files are mis-formatted, and throw an error if so. The user will need to format these files, whether through running `make format-sql`, VSCode auto-formatter, etc.